test_scripts/API/AutoCompleteList/003_ATF_SetGlobalProperties_N_transfer_NotAllowed_request.lua

```diff
@@ -0,0 +1,185 @@
+-----------------------------------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = false, resultCode = "DISALLOWED"> to mobile app
```

Comment:
@Aleksandriys, please update summary  - currently SDL is used twice: "..SDL SDL ..."